### PR TITLE
Improve the installation part in the readme file

### PR DESCRIPTION
**Pull request by delyriand**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +21,13 @@ Basically it disables the routes and updates the admin and frontend templates.
 ## Installation
 
 ```bash
-composer require monsieurbiz/sylius-no-commerce-plugin="1.0.x-dev" --no-scripts
+composer require monsieurbiz/sylius-no-commerce-plugin --no-scripts
```
Comment (jacquesbh) [inline README.md]:
```suggestion
composer require monsieurbiz/sylius-no-commerce-plugin
```

File: README.md [hunk 2]
```diff
@@ -21,11 +21,13 @@ Basically it disables the routes and updates the admin and frontend templates.
...
 ```
 
-Change your `config/bundles.php` file to add the line for the plugin : 
+<details><summary>For the installation without flex, follow these additional steps</summary>
```
Comment (jacquesbh) [inline README.md:27]: Don't forget to explain the `--no-scripts` here, for composer.

File: README.md [hunk 3]
```diff
@@ -35,12 +37,20 @@ return [
...
 
 ```yaml
 imports:
     - { resource: "@MonsieurBizSyliusNoCommercePlugin/Resources/config/config.yaml" }
+```
+
+</p>
+</details>
 
+In the config file `config/packages/monsieurbiz_sylius_nocommerce_plugin.yaml`, add this lines:
```
Comment (jacquesbh) [inline README.md]: Finally not in installation step, but more in customization?

**Review by jacquesbh — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (delyriand) [reply README.md:27]: with the fix for the default configuration, this option is useless

**Review by jacquesbh — APPROVED** [review]

**Review by lanfisis — APPROVED** [review]

**Comment by jacquesbh** [comment]
Thank you @delyriand !

**Pull request merged**